deploy/examples/collector.yaml

```diff
@@ -0,0 +1,79 @@
+    clusterName: openshift-demo-cluster
```

Comment:
Does this differ from the existing example in anyway?

Reply:
Existing example has configMap metadata and proxy address that would not work in Openshift.

Reply:
Ah I see. Can you rename this file to `openshift-config.yaml` and add a comment at the top of the file stating this is a plain yaml file shipped by default in openshift containers.

Reply:
Done.